### ISSUE TITLE
Summary metrics

### ISSUE THREAD

**Issue by ewildgoose** [mobius-home/mobius#47]
I see there is initial implementation of the summary() metric, however, it's not currently possible to plot these values. (Or I'm unable to figure out how it could be done?)

I'm pondering how we could implement this, my thought is as follows:

- Use the last dotted piece of the metric name as the index into the summary structure

So say we had a metric definition 

```
metrics = [ 
  summary("vm.memory.total") 
]
```

We would then call plot with something like

```
Mobius.plot("vm.memory.total.max", %{}, [type: summary])
```

This can be implemented in our filter function, if the type is :summary, then we unshift off the last dotted piece of the metric  name for our search ("max" in this example), then use it to select the inner metric to return

What do you think?

**Comment by simonprev** [comment]
This feature already seems implemented in `mobius` 🎉 

Maybe I’m missing something but with

```elixir
Mobius.Exports.plot "phoenix.router_dispatch.stop.duration", {:summary, :average}, %{}
```

The average just builds up overtime so as the time goes by, the metric will have less "impact" on the plot

What do you think @ewildgoose ?

**Comment by ewildgoose** [comment]
Yes, this can be closed as I already implemented it in pull request #69
